Software/Development/SemesterProject/App/Include/dutyCycle.h

```diff
@@ -30,24 +35,43 @@
 typedef struct DutyCycleTag
 {
     float dutyValue;
+    float minDutyValue;
+    float maxDutyValue;
```

Comment:
As discussed, maybe this minmax values would fit better in dutyCycleList.

Reply:
I will work on it!